cmake/projects/NASM/hunter.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, Zhuhao Wang
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+if(CMAKE_HOST_WIN32)
+  # NASM only supports a subset of VS compilers on Windows. However, the only
```

Comment:
It's okay but in general it's preferable to use sources, in case somebody want to patch this package.

Just for your info the feature request when we want to mix different toolchain in one build (e.g. use `nasm` from root with Visual Studio build in MinGW) is here:
* https://github.com/ruslo/hunter/issues/495